### PR TITLE
<details>,<dfn>,<diolog>,<div> and <dl> elements improved

### PR DESCRIPTION
**Pull request by Mehrm0**
### Description

I modified \<details\>,\<dfn\>,\<diolog\>,\<div\> and \<dl\> elements

### Motivation

According to MDN's documentations,  live samples should be preceded by a *Result* title, I added missing titles and I also inserted some missing live sample macros.

### CODE REVIEW

File: files/en-us/web/html/element/dfn/index.md [hunk 1]
```diff
@@ -100,8 +98,6 @@ Here we see the definition — now with an [`id`](/en-US/docs/Web/HTML/Global_at
 
 #### Result
```
Comment (CBID2) [inline files/en-us/web/html/element/dfn/index.md:99]:
```suggestion
#### Results
```

File: files/en-us/web/html/element/dl/index.md [hunk 2]
```diff
@@ -119,6 +121,8 @@ This element only includes the [global attributes](/en-US/docs/Web/HTML/Global_a
 </dl>
 ```
 
+#### Result
+
 {{EmbedLiveSample("Multiple_terms_single_description")}}
 
 ### Single term, multiple descriptions
```
Comment (CBID2) [inline files/en-us/web/html/element/dl/index.md:128]:
```suggestion
### Single term & multiple descriptions
```

**Review by CBID2 — COMMENTED** [review]
Hi @Mehrm0!😄 I just made one suggestion to improve your PR. Other than that, well done! 😄 👍🏾

File: files/en-us/web/html/element/dl/index.md (hunk 2 above)
Comment (wbamberg) [reply files/en-us/web/html/element/dl/index.md:128]: Thank you for the suggestion! But the title is the original (it wasn't changed in this PR), and I think the comma is OK here, and whatever we choose needs to match the macro call, so let's leave it, in the interests of merging this PR.

**Review by wbamberg — APPROVED** [review]
👍 thank you @Mehrm0 !

File: files/en-us/web/html/element/dl/index.md (hunk 2 above)
Comment (CBID2) [reply files/en-us/web/html/element/dl/index.md:128]: Ok

**Comment by Mehrm0** [comment]
> 



> Hi @Mehrm0!smile I just made one suggestion to improve your PR. Other than that, well done! smile 👍🏾
Thanks, it seems  it was merged before I could apply suggestions.

**Pull request merged**